### PR TITLE
Addition of T&C changes

### PR DESCRIPTION
**Pull request by jh-bate**
@darinkrauss if you could take a look at this. Obviously the API isn't what we want yet but just wanted to get the docs of what we have now there 

- updates that has come in with the T&C changes and examples generated via postman.
- some basic cleanup around other user management functions including examples

### CODE REVIEW

File: _docs/tidepool-api/manage-user.md [hunk 1]
```diff
@@ -65,6 +143,37 @@ Response:
 ```
 
 
+### Update user
+
+Apply updates to an existing users account for any or all of the feilds listed below
```
Comment (darinkrauss) [inline _docs/tidepool-api/manage-user.md]: fields (sp)

File: _docs/tidepool-api/manage-user.md [hunk 2]
```diff
@@ -10,6 +10,71 @@ For authentication, maintaining session token and logging out, see the introduct
...
+Then, create the **profile**:
+
+```
+POST /metadata/:userid/profile
+x-tidepool-session-token: <token>
+```
+
+Example Request:
+```
+curl -X POST -H "Content-Type: application/json" -H "x-tidepool-session-token: <token>" -d '{"fullName": "Mary Smith", "shortName":"Mary", "patient": { "birthday": "1990-01-31", "diagnosisDate": "1999-01-31", "aboutMe": "I like oranges"}}' '<api-endpoint>/metadata/<userid>/profile'
```
Comment (darinkrauss) [inline _docs/tidepool-api/manage-user.md]: Drop the "aboveMe".  I'd like to lock down our schema so we don't get a bunch of stuff we haven't approved.  Our docs should eventually detail where custom data like this should go.

File: _docs/tidepool-api/manage-user.md [hunk 3]
```diff
@@ -10,6 +10,71 @@ For authentication, maintaining session token and logging out, see the introduct
...
+```
+
+```json
+{
+  "fullName": "Mary Smith",
+  "shortName":"Mary",
+  "patient": {
+    "birthday": "1990-01-31",
+    "diagnosisDate": "1999-01-31",
+    "aboutMe": "I like oranges"
```
Comment (darinkrauss) [inline _docs/tidepool-api/manage-user.md]: Drop "aboutMe".

**Comment by darinkrauss** [comment]
A couple comments, but otherwise LGTM.

**Comment by jh-bate** [comment]
done

**Pull request merged**